chapters/parallel/index.md

```diff
@@ -0,0 +1,66 @@
+## Parallelization in Jina
+
+##### Learn about how Jina allows for parallelization of data processing
+
+### Feature description and expected outcome
+
+Jina allows for parallelization of Peas and Pods. This generally leads to an increase in throughput, as the data is split across processes. However, depending on the configuration, this can also lead to 
+
+### Before you start
+
+You are expected to be familiar with the general concepts of Jina. You should also have experience with parallelism in other systems. Jina does follow the best practices and designs from other databases or search engines. However, Jina is primarily a *neural search engine*. This means some changes and adaptations were necessary. This chapter will explain these.
+
+### Implementation
+
+We will approach this from a top-down perspective. 
+
+#### Inter-parallelism
+
+`Pods` are the highest level of abstraction where we can have parallelism. This is achieved by using the `needs` parameter.
+
+```python
+f = (Flow().add(name='p1', needs='gateway')
+           .add(name='p2', needs='gateway')
+           .add(name='p3', needs='gateway')
+           .needs(['p1','p2', 'p3'], name='r1').plot())
+```
+
+The plot of the above topology looks like this:
+
+![](./simple-plot3.svg)
+
+In the above, `p1`, `p2`, `p3` now subscribe to `gateway` and conduct their work in parallel. The last `.needs()` blocks all Pods until they finish their work.
+
+#### Intra-parallelism
+
+Peas are the next level at which we can configure parallelism. In this case, `Pod` receives the data (in its `HeadPea`) and then distributes it to its `Peas`. You can achieve this by using the `parallel` parameter:
```

Comment:
Unbacktick anything that isn't text you see in CLI input/output or code (e.g. Pod, Peas)

@FionnD Do we have guidance on this in style guide? I see a lot of inconsistency and room for confusion (to be honest, with myself also)

Reply:
https://developers.google.com/style/text-formatting says code formatting should be added for class names (Peas) and variables (parallel)

Reply:
- `parallel` - no issue with that :smile: 
- Peas/Pods - I thought you were talking about the things themselves, not the class names. If talking class names, keep the backticks and ignore my prior comment!